src/jats/__tests__/jats-journal-meta.test.ts

```diff
@@ -36,7 +36,7 @@ describe('JATS journal metadata', () => {
       abbreviatedTitle: 'PubMed Abbreviated Journal Title',
     })
     expect(journal.ISSNs).toHaveLength(1)
-    expect(journal.ISSNs![0]).toStrictEqual({ ISSN: '1234-5678' })
+    expect(journal.ISSNs![0]).toEqual({ ISSN: '1234-5678' })
```

Comment:
Why is that changed to toEqual? Do we expect ISSN to be undefined in some cases?

Reply:
Changes to the parsing code resulted in the parsed ISSN to have an undefined "publicationType" field. The ISSN itself will not be undefined.